scripts/CsvToJson.js

```diff
@@ -34,10 +39,12 @@ class CsvToJsonScript extends Base {
 
     // Execute external CSV to JSON module
     exec(
-      `${csvToJsonPath} ${sourceFilepath} > ${outputFilepath}`,
+      `${csvToJsonPath} ${sourceFilepath} > ${outputFilepath} ${moduleParams}`,
       (err, stdout, stderr) => {
         if (err) throw err;
-        console.log(`stdout: ${stdout}`, `stderr: ${stderr}`);
+        if (stdout) console.log(`stdout: ${stdout}`);
+        if (stderr) console.log(`stderr: ${stderr}`);
+        console.log('CSV to JSON data conversion complete.');
```

Comment:
I've found https://github.com/chalk/chalk to be quite useful when doing messages like that. Allows you to indicate statuses by coloring the console output. Small change, but useful. Out of the scope of this PR, but have a look if it could be useful for you.

Reply:
Great call, will look to implement this. Added to backlog 👍